lib/seed.js

```diff
@@ -170,7 +170,10 @@ function getConnection() {
       connectionString = 'mongodb://' + config[NODE_ENV]
     }
 
-    mongo.connect(connectionString, (err, client) => {
+    mongo.connect(connectionString, {
+      useUnifiedTopology: true,
+      useNewUrlParser: true
+    }, (err, client) => {
```

Comment:
Can you link to the docs for these? You've added them as hard coded values and no options to change them. Someone might not want these values.